src/commands/initProjectForVSCode/InitVSCodeStep/InitVSCodeStepBase.ts

```diff
@@ -189,21 +189,29 @@ export abstract class InitVSCodeStepBase extends AzureWizardExecuteStep<IProject
         return existingConfigs;
     }
 
-    private async writeSettingsJson(folder: WorkspaceFolder | undefined, vscodePath: string, language: string, version: FuncVersion): Promise<void> {
+    private async writeSettingsJson(context: IProjectWizardContext, vscodePath: string, language: string, version: FuncVersion): Promise<void> {
         const settings: ISettingToAdd[] = this.settings.concat(
             { key: projectLanguageSetting, value: language },
             { key: funcVersionSetting, value: version },
             // We want the terminal to be open after F5, not the debug console (Since http triggers are printed in the terminal)
             { prefix: 'debug', key: 'internalConsoleOptions', value: 'neverOpen' }
         );
 
+        // Add "projectSubpath" setting if project is far enough down that we won't auto-detect it
+        if (path.posix.relative(context.projectPath, context.workspacePath).startsWith('../..')) {
```

Comment:
So since a folder that's 5 subfolders deep from the root technically starts '../...', would this code detect those cases too?  Is there any concern about performance?

Reply:
Discussed offline